src/main/java/com/lgtm/easymoney/controllers/RequestController.java

```diff
@@ -25,11 +22,36 @@ public RequestController(UserService userService, RequestService requestService)
         this.requestService = requestService;
     }
 
+
     @PostMapping("/create")
-    // make a new request
     public ResponseEntity<RequestRsp> createARequest(@Valid @RequestBody RequestReq req) {
         return requestService.createARequest(req);
     }
-    // TODO handle other request routes
+
+    @GetMapping("/get/{uid}")
```

Comment:
Why not "/{uid}" so that the complete path can be "/request/{uid}" as we confirmed previously in the FlowChart and API Doc